samcli/local/events/api_event.py

```diff
@@ -191,7 +204,10 @@ def to_dict(self):
                      "resource": self.resource,
                      "requestContext": request_context_dict,
                      "queryStringParameters": dict(self.query_string_params) if self.query_string_params else None,
+                     "multiValueQueryStringParameters": dict(self.multi_value_query_string_params)
```

Comment:
do we need ```dict(self.multi_value_query_string_params)``` again? we do validation above to make sure its a dict.

Reply:
Probably because someone could mutate the public variable on the object after initializing?